bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymer-simple-slider",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
@freakypie All libs on Polymer v1+ normally have a major version bump (ie. `1.0.0`)
Food for thought

cc: @ruyadorno Could you expedite this pull request??